app/admin/user_addresses.rb

```diff
@@ -0,0 +1,19 @@
+ActiveAdmin.register UserAddress do
+  belongs_to :user
+  permit_params :user_id, :fullname, :address_line1, :address_line2, :city,
```

Comment:
```:user_id``` can be removed since belongs_to already adds this to permit_params.

reference: https://github.com/Fivell/activeadmin/blob/28891b58716ba3348d47447f085d88bbdea29c4d/lib/active_admin/resource_dsl.rb#L71